schemas/treatment.json

```diff
@@ -178,6 +178,30 @@
         "core": true,
         "displayName": "Response to Therapy"
       }
+    },
+    {
+      "name": "clinical_trials_database",
+      "description": "",
+      "valueType": "string",
+      "meta": { "display name": "Clinical Trials Database" },
+      "restrictions": {
+        "codeList": [
+           "NCI Clinical Trials",
+           "EU Clinical Trials Register"
+        ]
+      }
+    },
+    {
+      "name": "clinical_trial_number",
+      "description": "",
```

Comment:
Suggested description:

Based on the clinical_trial_database, indicate the unique clinical trial identifier of which the donor is a participant.